nima/tests/integration/http2/client/src/test/java/io/helidon/nima/tests/integration/http2/client/Http2ClientTest.java

```diff
@@ -35,19 +35,17 @@
 import io.helidon.nima.webserver.WebServer;
 import io.helidon.nima.webserver.http.HttpRouting;
 
-import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.MatcherAssert.assertThat;
 
 @ServerTest
 class Http2ClientTest {
-    public static final String MESSAGE = "Hello World!";
+     static final String MESSAGE = "Hello World!";
```

Comment:
there seems to be a different number of spaces for the package local constant than for private constants

Reply:
how did that ... fixed